docs/Rootless.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Rootless
+layout: docs
+---
+
+### Context
+
+Practically all jailbreaks prior to iOS 15 have been 'rootful' as they work on and install files to the (root) system directories (e.g., `/usr`, `/Library`, `/Applications`, etc). Moving forward with iOS 15 and beyond, most if not all jailbreaks will have to be 'rootless' as Apple now prevents writing to the system directories with a protection called [Signed System Volume (SSV)](https://support.apple.com/guide/security/signed-system-volume-security-secd698747c9/web). There are workarounds for this, namely bindFS, but these are less than ideal.
+
+To work around SSV, the [Procursus Team](https://github.com/procursusteam/) has thought up a rootless scheme that is now the de facto rootless standard. The changes namely involve working in and installing tweaks/addons to `/var/jb` and using the `iphoneos-arm64` package architecture. To learn more about the specifics of this implementation, see [The Apple Wiki](https://theapplewiki.com/wiki/Rootless).
+
+With this new target location, *all* projects will have to be recompiled to reference resources from and install to `/var/jb`. This involves changing the internal structure of .deb's to include this new path and specifying library and framework install paths using `@rpath` instead of absolute paths. Theos will handle both of these changes for you. For more information on what `@rpath` is and how it works, see Mike Ash's [informative blog post](http://www.mikeash.com/pyblog/friday-qa-2009-11-06-linking-and-install-names.html).
+
+### Implementation
+
+---
+
+**Important Notes**:
+- You must run `make clean` when switching between a rootful and rootless build
+
+- If you maintain a library, you'll want to add some variation of the following to your code so that it will have the correct install path:
+```make
+ifeq ($(THEOS_PACKAGE_SCHEME),rootless)
+XXX_LDFLAGS += -install_name @rpath/<lib-name>.dylib
+endif
+```
+
+- Likewise, if you maintain a framework, you'll want to add some variation of the following to your code so that it will have the correct install path:
+```make
+ifeq ($(THEOS_PACKAGE_SCHEME),rootless)
+XXX_LDFLAGS += -install_name @rpath/<project-name>.framework/<project-name>
+endif
+```
+
+- The iOS 14 arm64e ABI mentioned in [arm64e Deployment](arm64e-Deployment.html) is now *required* for the relevant devices
```

Comment:
This should probably be expanded with some more details:
* It's currently only possible to build for the new arm64e ABI on macOS. Developers who only have access to Linux may depend on the `oldabi` package instead, but that should preferably only be used as a last resort.
* If you still want to support iOS 13 and below, you'll want to build with Xcode 11.7 for rootful and 12.0+ for rootless. It is enough to take just the toolchain from Xcode 11.7, the rest can be deleted. Here is a build script that can be used assuming the legacy toolchain has been copied into `$THEOS/toolchain/Xcode.arm64eLegacy.xctoolchain`:
  ```sh
  #!/bin/sh -e

  threads=$(sysctl -n hw.logicalcpu)

  echo '[*] Building rootful package...'
  PREFIX=$THEOS/toolchain/Xcode.arm64eLegacy.xctoolchain/usr/bin/ gmake -j"$threads" clean package FINALPACKAGE=1

  echo

  echo '[*] Building rootless package...'
  gmake -j"$threads" clean package FINALPACKAGE=1 THEOS_PACKAGE_SCHEME=rootless
  ```

Reply:
Good call!